### PR TITLE
Adding atan2 key to maths calc and and also expanding the expression on the calculator

### LINKED ISSUE #5333: Two very small additions to the calculator
The base R has the trigonometric functions we have in our calculator in the maths keyboard.  See [here](https://stat.ethz.ch/R-manual/R-devel/RHOME/library/base/html/Trig.html).

Well the keyboard has all except one.  Please could we add the **atan2** key.  The key can say **atan2**.

While making this change a second small change is to make the expression field wider.  I am often not able to show the expression in the field.  Could it go up to **m in arguments**?

### PR DESCRIPTION
**Pull request by Wycklife**
Fixes #5333 

@africanmathsinitiative/developers This is ready for review.

### CODE REVIEW

**Comment by Carolnikie** [comment]
I've reviewed and it looks good from my side

File: instat/ucrCalculator.vb [hunk 1]
```diff
@@ -1162,7 +1169,7 @@ Public Class ucrCalculator
 
     Private Sub cmdNonMiss_Click(sender As Object, e As EventArgs) Handles cmdNonMiss.Click
         If chkShowArguments.Checked Then
-            ucrReceiverForCalculation.AddToReceiverAtCursorPosition("sum(!is.na(x= ))", 2)
+            ucrReceiverForCalculation.AddToReceiverAtCursorPosition("sum(!Is.na(x= ))", 2)
```
Comment (dannyparsons) [inline instat/ucrCalculator.vb]: Why has `is` changed to `Is` in several places? @Carolnikie did you notice this?

File: instat/ucrCalculator.vb [hunk 2]
```diff
@@ -1135,17 +1135,24 @@ Public Class ucrCalculator
             ucrReceiverForCalculation.AddToReceiverAtCursorPosition("stats::plogis()", 1)
         End If
     End Sub
+    Private Sub cmdAtan2_Click(sender As Object, e As EventArgs) Handles cmdAtan2.Click
+        If chkShowArguments.Checked Then
+            ucrReceiverForCalculation.AddToReceiverAtCursorPosition("atan2(y, x)", 1)
```
Comment (dannyparsons) [inline instat/ucrCalculator.vb]: There are no `=` signs after `y` and `x` but there are for other keys, should these be consistent?

Comment (shadrackkibet) [reply instat/ucrCalculator.vb]: yes, this should be consistent. I remember we maintained `=`  for all other keys.

Comment (dannyparsons) [reply instat/ucrCalculator.vb]: That sounds sensible and I suggest we also use the convention of spaces around `=` 
i.e. `x = 10` instead of `x=10`.

**Comment by Wycklife** [comment]
@dannyparsons I've made the changes.

**Comment by dannyparsons** [comment]
Please check the file changes, there are more cases of `is` becoming `Is`.

File: instat/ucrCalculator.vb (hunk 2 above)
Comment (shadrackkibet) [reply instat/ucrCalculator.vb]: I like this convention. Good coding practice.

File: instat/ucrCalculator.vb [hunk 3]
```diff
@@ -1152,7 +1152,7 @@ Public Class ucrCalculator
     End Sub
 
     Private Sub cmdNotIsNa_Click(sender As Object, e As EventArgs) Handles cmdNotIsNa.Click
-        ucrReceiverForCalculation.AddToReceiverAtCursorPosition("!Is.na()", 1)
+        ucrReceiverForCalculation.AddToReceiverAtCursorPosition("!is.na()", 1)
```
Comment (shadrackkibet) [inline instat/ucrCalculator.vb]: There are two other cases similar to this.

File: instat/ucrCalculator.vb [hunk 4]
```diff
@@ -1135,17 +1135,24 @@ Public Class ucrCalculator
...
     End Sub
 
     Private Sub cmdIsNa_Click(sender As Object, e As EventArgs) Handles cmdIsNa.Click
-        ucrReceiverForCalculation.AddToReceiverAtCursorPosition("is.na()", 1)
+        ucrReceiverForCalculation.AddToReceiverAtCursorPosition("Is.na()", 1)
     End Sub
 
     Private Sub cmdNotIsNa_Click(sender As Object, e As EventArgs) Handles cmdNotIsNa.Click
-        ucrReceiverForCalculation.AddToReceiverAtCursorPosition("!is.na()", 1)
+        ucrReceiverForCalculation.AddToReceiverAtCursorPosition("!Is.na()", 1)
```
Comment (shadrackkibet) [inline instat/ucrCalculator.vb]: here

File: instat/ucrCalculator.vb [hunk 5]
```diff
@@ -1135,17 +1135,24 @@ Public Class ucrCalculator
...
+        End If
+    End Sub
 
     Private Sub cmdDuplicate_Click(sender As Object, e As EventArgs) Handles cmdDuplicate.Click
         ucrReceiverForCalculation.AddToReceiverAtCursorPosition("duplicated()", 1)
     End Sub
 
     Private Sub cmdIsNa_Click(sender As Object, e As EventArgs) Handles cmdIsNa.Click
-        ucrReceiverForCalculation.AddToReceiverAtCursorPosition("is.na()", 1)
+        ucrReceiverForCalculation.AddToReceiverAtCursorPosition("Is.na()", 1)
```
Comment (shadrackkibet) [inline instat/ucrCalculator.vb]: here

**Comment by Wycklife** [comment]
@dannyparsons I've added the suggested keys to Dates calculator, and a minor change to the maths calculator.
@africanmathsinitiative/developers Someone can review these changes.

**Comment by maxwellfundi** [comment]
@Ivanluv @Ogik99 Please rereview this

**Comment by Ivanluv** [comment]
this is working well from my review

**Comment by dannyparsons** [comment]
The spacing around the `=` for the new date keys is not following the agreed format.

**Comment by Wycklife** [comment]
@dannyparsons  Could you please check if it's now okay

**Pull request merged**